internal/store/store.go

```diff
@@ -0,0 +1,11 @@
+package store
+
+import "context"
+
+type Image struct {
+	Reference string
+}
+
+type Storer interface {
```

Comment:
Store is a bad name but I could not come up with something else.

Reply:
idea dump ArtifactSourcer